status-im/status-react
# feat: implement address input

fixes #17189 

### Summary

Implement Address input

#### Screenshots

![](https://github.com/status-im/status-mobile/assets/18485527/31c141de-100d-435b-bd89-82b174c20297) ![](https://github.com/status-im/status-mobile/assets/18485527/fe501859-bb3c-4930-a80b-ed2db5d1dab7)

#### Platforms

- Android
- iOS

### Steps to test


- Open Status
- Open Quo2 previes
- Select Inputs > Address input
- Verify correct design and behavior of the component

status: ready
